mungegithub/mungers/blunderbuss.go

```diff
@@ -162,6 +162,19 @@ func selectMultipleOwners(potentialOwners weightMap, weightSum int64, count int)
 	return owners
 }
 
+func getPotentialOwnersHelper(author *string, feats *features.Features, files []*githubapi.CommitFile, issueNumber *int) (weightMap, int64) {
+	potentialOwners, weightSum := getPotentialOwners(*author, feats, files, true)
+	if len(potentialOwners) == 0 {
```

Comment:
I mean rewriting it somewhat like this:

```
func getPotentialOwnersHelper(author *string, feats *features.Features, files []*githubapi.CommitFile, issueNumber *int) (weightMap, int64) {
 	potentialOwners, weightSum := getPotentialOwners(*author, feats, files, true)
 	if len(potentialOwners) != 0 {
                return potentialOwners, weightSum
        }

        potentialOwners, weightSum = getPotentialOwners(*author, feats, files, false)
 	if len(potentialOwners) != 0 {
 		return potentialOwners, weightSum
 	}
 
	glog.Errorf("No OWNERS found for PR %d", *issueNumber)
 	return nil, 0
 }
```

But it doesn't really matter. I'm just not a big fan of the nested `if`s